plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -173,7 +183,8 @@ public List<HostAddress> getAddresses()
 
     public List<String> getAllSchemas()
     {
-        return ImmutableList.copyOf(client.listDatabaseNames()).stream()
+        return ImmutableList.copyOf(listDatabaseNames())
```

Comment:
Rollback. `stream` in the same line is typical in Trino.
And BTW, always do those kind of changes in a different commit.